diff --git a/src/EventTrackerService.php b/src/EventTrackerService.php
index 4689f8f..c037d82 100644
--- a/src/EventTrackerService.php
+++ b/src/EventTrackerService.php
@@ -321,6 +321,17 @@ class EventTrackerService {
       $this->eventDispatcher->dispatch(EnhancedEcommerceEvents::ALTER_PRODUCT_PURCHASED_ENTITY, $event);
     }
 
+    // Update price from order item.
+    $total_price = $order_item->getTotalPrice();
+    if ($total_price) {
+      $amount = (float) $total_price->getNumber();
+      $price = self::formatPrice($amount);
+    }
+    else {
+      $price = self::formatPrice(0);
+    }
+    // Update price.
+    $product->setPrice($price);
     return $product;
   }
 
