diff --git a/src/Plugin/Condition/Licensing.php b/src/Plugin/Condition/Licensing.php
index 307ef5a..ee62d55 100644
--- a/src/Plugin/Condition/Licensing.php
+++ b/src/Plugin/Condition/Licensing.php
@@ -88,6 +88,10 @@ class Licensing extends EntityBundle implements ConstraintConditionInterface, Co
    *   TRUE if the condition has been met, FALSE otherwise.
    */
   public function evaluate() {
+    $bundles = $this->configuration['bundles'] ?? [];
+    if (empty($bundles)) {
+      return TRUE;
+    }
     /** @var \Drupal\Core\Session\AccountInterface $active_user */
     $active_user = $this->getContextValue('user');
 
